pom.xml

```diff
@@ -1028,6 +1028,54 @@
             </executions>
           </plugin>
           <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <version>3.0.0</version>
+              <executions>
+                  <!-- TODO how to avoid downloading them all? -->
+                  <execution>
+                      <id>download node for Linux</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:tar.gz:linux-x64</artifact>
+                      </configuration>
+                  </execution>
+                  <execution>
+                      <id>download node for Darwin</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:tar.gz:darwin-x64</artifact>
+                      </configuration>
+                  </execution>
+                  <execution>
+                      <id>download node for Windows</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:exe:windows-x64</artifact>
```

Comment:
If someone actually needs it, we can upload it, but I doubt anyone is using it anyway.